.github/workflows/weekly.yml

```diff
@@ -37,10 +37,6 @@ jobs:
           flink: 1.19-SNAPSHOT,
           jdk: '8, 11, 17, 21',
           branch: main
-        }, {
-          flink: 1.18-SNAPSHOT,
-          jdk: '8, 11, 17',
-          branch: main
```

Comment:
why do we remove 1.18 here and do not do that for pr flow?

Reply:
Good catch! Will remove from pr flow